detailed_search_autonomy.py

```diff
@@ -20,7 +20,7 @@
 # Callback function for messages from GCS, parses JSON message and sets globals
 def xbee_callback(message, autonomyToCV):
     address = message.remote_device.get_64bit_addr()
-    msg = json.loads(message.data.decode("utf8"))
+    msg = json.loads(message.data)
```

Comment:
What changed here? why aren't we decoding this anymore?

Reply:
Python3 does not require .decode() anymore